src/Raven.Server/Documents/Replication/ReplicationLoader.cs

```diff
@@ -367,7 +367,7 @@ private ReplicationLatestEtagRequest IncomingInitialHandshake(TcpConnectionOptio
                 using (documentsOperationContext.OpenReadTransaction())
                 using (configurationContext.OpenReadTransaction())
                 {
-                    var changeVector = DocumentsStorage.GetDatabaseChangeVector(documentsOperationContext);
+                    var changeVector = DocumentsStorage.GetFullDatabaseChangeVector(documentsOperationContext);
```

Comment:
Shouldn't we default to the database change vector if empty?

Reply:
if full CV is empty so does regular database CV